frontend/packages/kubevirt-plugin/integration-tests/tests/utils/constants/enums/provisionSource.ts

```diff
@@ -0,0 +1,29 @@
+import { ObjectEnum } from '../../../../../src/constants/object-enum';
```

Comment:
can you please create a PR for moving object-enum to console/shared? So we don't have to depend on the kubevirt code base here? 

Or I can do the PR - up to you.

Reply:
ok